packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -32,7 +38,18 @@ Celo uses a [proof-of-stake](../celo-codebase/protocol/proof-of-stake) consensus
 
 Participating in the Baklava testnet requires testnet units of CELO, which can only be used in the Baklava testnet. You can request a distribution of testnet CELO by filling out [the faucet request form](https://forms.gle/JTYkMAJWTAUQp1sv9). If you need any help getting started, please join the discussion on [Discord](https://chat.celo.org) or email community@celo.org.
 
-Faucetted funds will come as two `ReleaseGold` contracts. At a high level, `ReleaseGold` holds a balance for scheduled release, while allowing the held balance to be used for certain actions such as validating and voting, depending on the configuration of the contract. [Read more about `ReleaseGold`.](../celo-holder-guide/release-gold.md).
+Fauceted funds come two different ways, depending on whether you're using ReleaseGold smart contract or you are validating the regular way. For every new validator, we encourage going the non-ReleaseGold route.
```

Comment:
typo: add "a" or pluralize "contracts":

Fauceted funds come two different ways, depending on whether you're using `a` ReleaseGold smart contract...

Fauceted funds come two different ways, depending on whether you're using ReleaseGold smart `contracts`

Reply:
typo: `Fauceted` vs `Faucetted`

Reply:
clarification for `you are validating the regular way`:
Does this mean that the `non-ReleaseGold` is the regular way?